site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -875,6 +903,124 @@ This strategy instructions FusionAuth when to append a unique suffix to the user
 * `Always` - Always append a unique suffix even when the requested username is not in use.
 * `OnCollision` - Only append a unique suffix when the requested username is in use.
 
+[field]#tenant.webAuthnConfiguration.bootstrapWorkflow.authenticatorAttachmentPreference# [type]#[String]# [optional]#Optional# [default]#defaults to `crossPlatform`# [since]#Available since 1.41.0#::
+Determines the authenticator attachment requirement for WebAuthn passkey registration when using the bootstrap workflow. The possible values are:
++
+ * `any` - An authenticator with any attachment modality will be allowed during registration.
+ * `crossPlatform` - Only authenticators with the `cross-platform` attachment modality will be allowed during registration. These are also referred to as "roaming" authenticators.
+ * `platform` - Only authenticators with the `platform` attachment modality will be allowed during registration.
++
+The recommended value for the bootstrap workflow is `crossPlatform` (when supported).
++
+:advanced_feature: WebAuthn
+include::../shared/_advanced-edition-blurb-api.adoc[]
++
+:advanced_feature!:
++
+:enterprise_feature: WebAuthn roaming authenticators
```

Comment:
```suggestion
:enterprise_feature: WebAuthn cross-platform authenticators
```

Reply:
The license features refer to them as "roaming" authenticators. @robotdan should we update that as well?

![image](https://user-images.githubusercontent.com/3409780/201677862-0b7f1815-93e8-4140-b22e-b16ce57ef4f1.png)

Reply:
Fixed the FusionAuth end.